cdk/lib/__snapshots__/cv-redact-tool.test.ts.snap

```diff
@@ -712,42 +714,6 @@ dpkg -i /cv-redact-tool/cv-redact-tool.deb",
       },
       "Type": "AWS::EC2::SecurityGroupEgress",
     },
-    "SSMRunCommandPolicy244E1613": {
-      "Properties": {
-        "PolicyDocument": {
-          "Statement": [
-            {
-              "Action": [
-                "ec2messages:AcknowledgeMessage",
-                "ec2messages:DeleteMessage",
-                "ec2messages:FailMessage",
-                "ec2messages:GetEndpoint",
-                "ec2messages:GetMessages",
-                "ec2messages:SendReply",
-                "ssm:UpdateInstanceInformation",
-                "ssm:ListInstanceAssociations",
-                "ssm:DescribeInstanceProperties",
-                "ssm:DescribeDocumentParameters",
-                "ssmmessages:CreateControlChannel",
-                "ssmmessages:CreateDataChannel",
-                "ssmmessages:OpenControlChannel",
-                "ssmmessages:OpenDataChannel",
-              ],
-              "Effect": "Allow",
-              "Resource": "*",
-            },
-          ],
-          "Version": "2012-10-17",
-        },
-        "PolicyName": "ssm-run-command-policy",
-        "Roles": [
-          {
-            "Ref": "InstanceRoleCvredacttool834C6B7C",
-          },
-        ],
-      },
-      "Type": "AWS::IAM::Policy",
-    },
```

Comment:
GuCDK v49.0.2 moves to using the AWS managed policy for SSM access - https://github.com/guardian/cdk/releases/tag/v49.0.2